crates/rune-modules/src/time.rs

```diff
@@ -29,53 +29,599 @@
 //! }
 //! ```
 
-use rune::{Any, ContextError, Module};
+use rune::{
+    docstring,
+    runtime::{Mut, VmError, VmResult},
+    vm_panic, vm_try, Any, ContextError, Module,
+};
```

Comment:
Imports are broken up per-module in the rest of the project. It leads to cleaner diffs.

Reply:
The current project style would be:

```rust
use rune:{docstring, vm_panic, Any, ContextError, Module};
use rune::runtime::{Mut, VmResult};
```